code/modules/clothing/clothing.dm

```diff
@@ -353,6 +353,10 @@ BLIND     // can't see anything
 	set src in usr
 	set_sensors(usr)
 
+/obj/item/clothing/under/AltClick(mob/user)
+	if(user.Adjacent(src))
+		set_sensors(user)
```

Comment:
I would consider moving the `if(user.Adjacent(src))` check inside the set_sensors proc

Reply:
Don't do this. You can move while the input box is up, so two checks is needed.

Reply:
Yeah, I didnt mean to remove it, just to move it to the beginning of the set_sensors proc, but thats just a suggestion